content/projects/gsoc/2023/project-ideas/alternative-jenkinsio-build-tool.adoc

```diff
@@ -43,6 +43,8 @@ While the tasks of the project are very clearly defined, the scope may vary depe
 
 The outcome of this project is expected to produce visible impact they can showcase in their portfolio of the link:https://www.jenkins.io/[jenkins.io] website, as the GSoC contributor is also expected to contribute via UI/UX improvements beyond the basic tooling required.
 
+Please note that for the UI/UX improvement portion we may need to deal with the link:https://github.com/jenkins-infra/jenkins-io-components[jenkins-io-components repo], where the code for the link:https://www.jenkins.io/[jenkins.io] website is currently hosted.
```

Comment:
```suggestion
Please note that for the UI/UX improvement portion we may need to deal with the link:https://github.com/jenkins-infra/jenkins-io-components[jenkins-io-components repo], where the code for components shared by various Jenkins websites (link:https://www.jenkins.io/[jenkins.io], link:https://plugins.jenkins.io/[plugins.jenkins.io], etc.) is currently hosted.
```

Reply:
Thanks for the suggested correction @zbynek